packages/rocketchat-integrations/client/startup.coffee

```diff
@@ -4,4 +4,4 @@ RocketChat.AdminBox.addOption
 	href: 'admin-integrations'
 	i18nLabel: 'Integrations'
 	permissionGranted: ->
-		return RocketChat.authz.hasAllPermission('manage-integrations')
+		return (RocketChat.authz.hasAllPermission 'manage-integrations') or (RocketChat.authz.hasAllPermission 'manage-own-integrations')
```

Comment:
Just a little code improvement, you can use `RocketChat.authz.hasAtLeastOnePermission(['manage-integrations', 'manage-own-integrations'])` when checking for permissions at client side.